sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_shared_key_policy_async.py

```diff
@@ -3,9 +3,13 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from __future__ import annotations
+from typing import TYPE_CHECKING
 import time
 import six
 from azure.core.pipeline.policies import SansIOHTTPPolicy
+if TYPE_CHECKING:
+    from azure.core.pipeline import PipelineRequest
 from ...aio._base_handler_async import ServiceBusSharedKeyCredential
```

Comment:
pylint error here for order of imports. Try the following:
```suggestion
from ...aio._base_handler_async import ServiceBusSharedKeyCredential
if TYPE_CHECKING:
    from azure.core.pipeline import PipelineRequest
```